### ISSUE TITLE
Developer tools panel

### ISSUE THREAD

**Issue by calebmer** [zalmoxisus/redux-devtools-extension#95]
Is there a way to give Redux a developer panel in the chrome developer tools? React developer tools has its own panel, is it possible for Redux to get one as well?

**Comment by zalmoxisus** [comment]
Thanks for the suggestion! It's easy to implement and I've even tested it before. But I'm not sure it would be right to have Redux DevTools there, as the `Elements` tab is intended to be used for DOM investigation, so React DevTools does.

I know it's not convenient to switch between `Elements` and `Redux` tab, but actually you can open Redux DevTools in a new window. Wouldn't this be easier to use?

**Comment by calebmer** [comment]
The developer tools has network investigation, performance profiling, security, and a code editor. I think a Redux tab would make sense 😊

It's more ergonomic for me to keep dev tools in the same window. For one that's how the tool designed (to be in the same window), and second on a Mac in fullscreen mode I can't have two windows on my monitor at once. It also allows me to use keyboard shortcuts to navigate around.

**Comment by zalmoxisus** [comment]
This should work:
![redux_router_with_autotodo_example_and_developer_tools_-_chrome-extension___lmhkpmbekcpmknklioeibfkpmmfibljd_background_html](https://cloud.githubusercontent.com/assets/7957859/14761065/daab14a8-095d-11e6-9b74-d5167f740924.jpg)

Don't you have the `Redux` tab?
I thought you want to have it inside the `Elements` tab, so you'll not have to switch between tabs.

**Comment by calebmer** [comment]
Yeah that's what I meant. And no I do not have that.

I'm on OSX with Chrome Beta (49.0.2623.112 (Official Build) beta (64-bit)) and Redux DevTools v1.0.1.

**Comment by zalmoxisus** [comment]
That's strange. I was thinking the problem is with the version from Chrome store, but tried it too and the tab is present. Could you please try to update Chrome? I'm using Chrome 50 on OSX, but on 49 it was working as well.

**Comment by calebmer** [comment]
Yep, upgraded to Chrome 51 and it works. A couple of other quick questions, is there a way to switch dev tools to a light theme, and is there a way to remove it from right clicks (or at least right clicks on non-activated pages)?

**Comment by zalmoxisus** [comment]
> Yep, upgraded to Chrome 51 and it works.
Except having a bug in Chrome beta, the only reason could be that you used a custom theme which tried to use the tabs icon. So, just in case I've added it.

> is there a way to switch dev tools to a light theme
That's a good idea. If you're interested in it, I've created [a separate issue](https://github.com/zalmoxisus/redux-devtools-extension/issues/96).

> is there a way to remove it from right clicks (or at least right clicks on non-activated pages)?
It was working like this in some older versions, but now we need the context menu also to open remote debugging, even if nothing is inspected in the browser pages.
